build/index.js

```diff
@@ -0,0 +1,229 @@
+const GAME_SIZE = 400
+const BLOCK_SIZE = 10
+const SNAKE_COLOR = 'green'
+const FOOD_COLOR = 'red'
+
+document.addEventListener('DOMContentLoaded', function() {
+  const snake = new Snake(4, 0, 0, BLOCK_SIZE, SNAKE_COLOR)
+
+  document.addEventListener('keydown', function(e) {
+    if (e.code === 'ArrowUp' || e.code === 'KeyW') {
+      snake.transitionUp()
```

Comment:
Delay actual velocity changes until the snake moves. This prevents issues where the user changes direction multiple times within one animation frame.